frontend/src/forms/subforms/EvaluationForm.tsx

```diff
@@ -1,90 +1,188 @@
-import { Fragment } from 'react';
+import { Fragment, useState } from 'react';
 import React from 'react';
-import { Col } from 'react-bootstrap';
-import { FormikProps } from 'formik';
+import { Col, Row, Table, InputGroup } from 'react-bootstrap';
+import { FormikProps, setIn, getIn } from 'formik';
 import { Form, Input } from 'components/common/form';
 import { IEvaluation } from 'actions/parcelsActions';
+import { EvaluationKeys } from 'constants/evaluationKeys';
+import moment from 'moment';
+import _ from 'lodash';
+import { FormikDatePicker } from 'components/common/FormikDatePicker';
+import WrappedPaginate from 'components/common/WrappedPaginate';
+import { IPaginate } from 'utils/CommonFunctions';
+import { FormikInputGroup } from 'components/common/form/InputGroup';
 
 interface EvaluationProps {
-  nameSpace?: string;
-  evaluation?: any;
-  index?: number;
+  /** the formik tracked namespace of this component */
+  nameSpace: string;
+  /** whether this form is enabled for editing */
   disabled?: boolean;
 }
-export const defaultEvaluationValues: IEvaluation = {
-  date: '',
-  key: '',
-  value: '',
+
+/**
+ * Extend IEvaluation, add the year of the evaluation (regardless of whether the date is set.)
+ */
+interface IFormEvaluation extends IEvaluation {
+  /** the year placeholder of this evaluation */
+  year?: number;
+}
+const NUMBER_OF_EVALUATIONS_PER_PAGE = 2;
+const NUMBER_OF_GENERATED_EVALUATIONS = 20;
```

Comment:
Weren't you going to set the furthest back date instead?  Otherwise in the future they won't see original values entered.

Reply:
In my memory there wasn't a resolution to that conversation (ie. a specific date) to lookback to (or even a specific number of years to lookback to.

Reply:
This will work for us now.